test_data/makedata_suites/021_foxx_utf-8.js

```diff
@@ -0,0 +1,150 @@
+
+      print("installing crud");
+      installFoxx(`/crud_${dbCount}`, minimalWorkingZip, "install", options);
+      db._useDatabase('_system');
+      return 0;
+    },
+    checkDataDB: function (options, isCluster, isEnterprise, database, dbCount, readOnly) {
+      let dbName = `${extendedNames[0]}FoxxTest${extendedNames[3]}_${dbCount}`;
+      db._useDatabase(dbName);
+      print(`checking data ${dbCount} ${dbName}`);
+      let reply;
+
+      [
+        aardvarkRoute,
+        `/_db/_system/itz_${dbCount}/index`,
+        `/_db/_system/crud_${dbCount}/xxx`
+      ].forEach(route => testFoxxReady(route));
+
+      print("Foxx: Itzpapalotl getting the root of the gods");
+      reply = arango.GET_RAW(`/_db/_system/itz_${dbCount}`);
+      assertEqual(reply.code, "307", JSON.stringify(reply));
+
+      print('Foxx: Itzpapalotl getting index html with list of gods');
+      reply = arango.GET_RAW(`/_db/_system/itz_${dbCount}/index`);
+      assertEqual(reply.code, "200", JSON.stringify(reply));
+
+      print("Foxx: Itzpapalotl summoning Chalchihuitlicue");
+      reply = arango.GET_RAW(`/_db/_system/itz_${dbCount}/Chalchihuitlicue/summon`, onlyJson);
+      assertEqual(reply.code, "200", JSON.stringify(reply));
+      let parsedBody = JSON.parse(reply.body);
+      assertEqual(parsedBody.name, "Chalchihuitlicue");
+      assertTrue(parsedBody.summoned);
+
+      print("Foxx: crud testing get xxx");
+      reply = arango.GET_RAW(`/_db/_system/crud_${dbCount}/xxx`, onlyJson);
+      assertEqual(reply.code, "200");
+      parsedBody = JSON.parse(reply.body);
+      assertEqual(parsedBody, []);
+
+      print("Foxx: crud testing POST xxx");
+
+      reply = arango.POST_RAW(`/_db/_system/crud_${dbCount}/xxx`, {_key: "test"});
+      if (options.readOnly) {
+        assertEqual(reply.code, "400");
+      } else {
+        assertEqual(reply.code, "201");
+      }
+
+      print("Foxx: crud testing get xxx");
+      reply = arango.GET_RAW(`/_db/_system/crud_${dbCount}/xxx`, onlyJson);
+      assertEqual(reply.code, "200");
+      parsedBody = JSON.parse(reply.body);
+      if (options.readOnly) {
+        assertEqual(parsedBody, []);
+      } else {
+        assertEqual(parsedBody.length, 1);
+      }
+
+      print('Foxx: crud testing delete document');
+      reply = arango.DELETE_RAW(`/_db/_system/crud_${dbCount}/xxx/` + 'test');
+      if (options.readOnly) {
+        assertEqual(reply.code, "400");
+      } else {
+        assertEqual(reply.code, "204");
+      }
+      db._useDatabase('_system');
+      return 0;
+    },
+    clearDataDB: function (options, isCluster, isEnterprise, database, dbCount) {
+      // All items created must contain dbCount
+      let dbName = `${extendedNames[0]}FoxxTest${extendedNames[3]}_${dbCount}`;
+      db._useDatabase(dbName);
+      print(`deleting foxx ${dbCount}${dbName}`);
+      print("uninstalling Itzpapalotl");
+      deleteFoxx(`/itz_${dbCount}`);
+
+      print("uninstalling crud");
```

Comment:
Do we really need this output?

Reply:
rta-makedata needs much output to track potential lockups.